agent_build/linux/scalyr-agent-2-config

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+scalyr-agent-2 config "$@"
```

Comment:
Depending on what we want to achieve, using ``exec`` may also be desired here to replace the script process.

Reply:
Ah sure, that should be a better approach. 

I guess it's also worth changing it to `sh`. I don't think that we'll add something complex here, that is only supported by bash. I am also not sure that the config command may be somehow required in docker images, but if so, our alpine images also would require a POSIX shell.

Reply:
Yeah, sh should be fine here since we are not doing anything complicated.